app/views/hiring_staff/vacancies/vacancy_form_partials/_heading.html.haml

```diff
@@ -1,5 +1,6 @@
 %h1.govuk-heading-m
-  = @vacancy.present? ? page_title(@vacancy) : t('jobs.create_a_job_title', organisation: current_organisation.name)
+  = @vacancy.present? ? page_title(@vacancy) : t('jobs.create_a_job_title_no_org')
   - unless %w(copy edit edit_published).include?(@vacancy&.state)
     %span.govuk-caption-m
       = t('jobs.current_step', step: current_step,  total: total_steps)
+      
```

Comment:
extra new lines at the end of the file, fix your editor so that we do not have this noise during PR review